src/main/scala/edu/colorado/plv/cuanto/scoot/ir/AddExpr.scala

```diff
@@ -0,0 +1,12 @@
+package edu.colorado.plv.cuanto.scoot.ir
```

Comment:
Let's call this package `scoot.jimple`. That is, by convention, `soot.jimple` parallels `scoot.jimple`.